pkg/detectors/typeform/v2/typeform.go

```diff
@@ -0,0 +1,96 @@
+package typeform
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(`\btfp_[a-zA-Z0-9_\.=\-]{40,59}\b`)
```

Comment:
The comment's suggestion to use a boundary character is actually bad advice, it won't match if the last character is `.=-`.

One possible alternative is to say "anything but":
```suggestion
	keyPat = regexp.MustCompile(`\btfp_[a-zA-Z0-9_\.=\-]{40,59}(?:[^a-zA-Z0-9_\.=\-]|\z)`)
```